p2p/discover/table.go

```diff
@@ -468,9 +467,7 @@ func (tab *Table) loadSeedNodes() {
 			addr, _ := seed.UDPEndpoint()
 			tab.log.Trace("Found seed node in database", "id", seed.ID(), "addr", addr, "age", age)
 		}
-		tab.mutex.Lock()
-		tab.handleAddNode(addNodeOp{node: seed, isInbound: false})
-		tab.mutex.Unlock()
+		go tab.handleAddNode(addNodeOp{node: seed, isInbound: false})
```

Comment:
does Go-Ethereum have the deadlock issue? seems like the lock tries to protect the `tab` object been accessed concurrently, start a new routine would break this protection, right?